tensorflow/python/data/ops/dataset_ops.py

```diff
@@ -3676,18 +3676,25 @@ class Options(options_lib.OptionsBase):
       default_factory=threading_options.ThreadingOptions)
 
   def __getattr__(self, name):
+    # handle backward compatibility with deprecated options
```

Comment:
you will still want to add `autotune` option attribute to `Options` so that it shows up in the API documentation ... (it will not be using by the implementation but that's an implementation detail)

Reply:
@jsimsa if we add the `autotune` option to `Options` then the `__getattr__()` function will not be called when we are trying to access `options.autotune`.